packages/files/file_path.pony

```diff
@@ -223,6 +223,9 @@ class val FilePath
         if info.directory and not info.symlink then
           0 == @_rmdir[I32](path.cstring())
         else
+          // _unlink() on Windows can't remove readonly files
+          // so we change mode to _S_IWRITE just in case
+          @_chmod[I32](path.cstring(), I32(0x0080))
```

Comment:
does this need to be done before 224 for directories as well?

Reply:
Oops, I put the `chmod()` in the wrong spot. Fixed.